### PR TITLE
[PHX-2023] Create event on DevOps Days site

### PR DESCRIPTION
**Pull request by desktophero**
Adding the Phoenix event to the website

### CODE REVIEW

**Comment by mattstratton** [comment]
Please send an email to info@devopsdays.org with the email addresses of your organizers listed on the Contact page and we will reconcile the `phoenix@devopsdays.org` email alias with that list (as well as ensure those folks are invited to slack). we will merge the PR when we receive that list!

File: data/events/2023-phoenix.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  # - id: samplesponsorname
+  #   level: gold
+ #  url: http://mysponsor.com/?campaign=me # Use this if you need to over-ride a sponsor URL.
+  # - id: arresteddevops
+  #   level: community
+
+sponsors_accepted : "no" # Whether you want "Become a XXX Sponsor!" link
```
Comment (mattstratton) [inline data/events/2023-phoenix.yml:73]: is this intentional? you do not want interested sponsors to contact you?

Comment (desktophero) [reply data/events/2023-phoenix.yml:73]: We do want sponsors to contact us. We're finishing up a small detail on the sponsors piece and need to reconcile the "who" they should contact ( a la your note above). 

I had the order of operations in my head to have it a "no" until contact info is updated and then flip to "yes". I'll update now.

Comment (mattstratton) [reply data/events/2023-phoenix.yml]: it's not a huge deal, i was just reviewing / second set of eyes :)

**Comment by mattstratton** [comment]
Email list updated and slack invites sent; this is clear to merge once the build is completed.

File: data/events/2023-phoenix.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+location: "Phoenix" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+  - name: registration
+ # - name: program
+  - name: speakers
```
Comment (mattstratton) [inline data/events/2023-phoenix.yml:42]: you'll want to comment this out until you have speakers; right now it makes a 404 if someone clicks on the nav element

File: data/events/2023-phoenix.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Phoenix" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+  - name: registration
```
Comment (mattstratton) [inline data/events/2023-phoenix.yml:40]: also might want to comment this out until the reg page is updated

Comment (mattstratton) [inline data/events/2023-phoenix.yml:40]:
```suggestion
  # - name: registration
```

File: data/events/2023-phoenix.yml (hunk 2 above)
Comment (mattstratton) [inline data/events/2023-phoenix.yml:42]:
```suggestion
  # - name: speakers
```

**Review by mattstratton — CHANGES_REQUESTED** [review]
I suggest commenting out these two items at the moment

**Pull request merged**